src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -54,10 +54,25 @@ public sealed class FormatHex : PSCmdlet
         [ValidateNotNullOrEmpty]
         public Encoding Encoding { get; set; } = ClrFacade.GetDefaultEncoding();
 
+        /// <summary>
+        /// Gets and sets count of bytes to read from the input stream from.
```

Comment:
Redundant `from` at the end of the sentence.

Reply:
Fixed.